### DISCUSSION TITLE
Next JS 13 -  How to enable the turbopack in new project

### DISCUSSION THREAD

**Discussion by MDmubarak786** [vercel/next.js#48240] [Help]
### Summary

I would like to setup the turbo pack in my new project. how do I can implement

**Comment by haydenbleasel** [comment]
Hey @MDmubarak786, should be as simple as opening your `package.json` file and changing `next dev` to `next dev --turbo`. More details here: https://nextjs.org/docs/advanced-features/turbopack

**Reply by MDmubarak786** [reply]
Thanks @haydenbleasel, Yes for new project is very easy. But how can we implement in existing project.

**Reply by Albertyhu** [reply]
You can follow this guide here on how to add Turbo pack to an existing Next JS project:

https://turbo.build/repo/docs/getting-started/add-to-project

**Reply by un33k** [reply]
@Albertyhu The title it is talking about Turbopack and not Turborepo.

**Reply by cy-alonzo** [reply]
I tried following the steps from the docs for existing project, but it says `error: unknown option --turbopack`.  I believe I need to install it, but how?

**Reply by theNodeMan** [reply]
> I tried following the steps from the docs for existing project, but it says `error: unknown option --turbopack`. I believe I need to install it, but how?

--turbo: https://nextjs.org/docs/app/api-reference/next-config-js/turbo
--turbopack: https://nextjs.org/docs/architecture/turbopack

--turbo works for me, but (NextJS 14.2.14) I get this error:

 ⚠ Webpack is configured while Turbopack is not, which may cause problems.
 ⚠ See instructions if you need to configure Turbopack:
  https://nextjs.org/docs/app/api-reference/next-config-js/turbo
